Vagrantfile

```diff
@@ -158,6 +158,7 @@ Vagrant.configure('2') do |config|
       cfg.vm.provision :chef_client do |chef|
         chef_defaults chef, symbol
         chef.add_recipe 'cerner_splunk::cluster_slave'
+        # chef.add_recipe 'cerner_splunk_test::configure_guids' # Uncomment this line if you want to set predefined GUIDs to you cluster slaves.
```

Comment:
Nit: I'd move the comment above the line so that way it's clear what the line is for instead of having to scroll right and say "Uncomment the line below to set predefined GUIDs on the cluster slaves (for playing with license pooling)"